robotoff/redis.py

```diff
@@ -39,4 +44,10 @@ def __enter__(self):
 
     def __exit__(self, *args, **kwargs):
         if self._enabled:
-            self.release()
+            try:
+                self.release()
+            except NotAcquired:
+                logger.info(
+                    "releasing lock failed, lock %s does not exist or expired",
+                    self._name,
+                )
```

Comment:
I think it's worth tracking those kind of errors (they should not happen too much, otherwise we really have to extend the time to live). So I would, at least, use logger.warning

Reply:
Good point, I've increased it to 5 min + replaced info by warning.